buildSrc/src/main/java/com/microsoft/hydralab/compile/UMLImageGenerator.java

```diff
@@ -0,0 +1,31 @@
+package com.microsoft.hydralab.compile;
```

Comment:
The copyright info is required for each new file.
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

Reply:
```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
package com.microsoft.hydralab.compile;
```